Test/monomorphize/monomorphize1.bpl

```diff
@@ -1,34 +1,13 @@
 
-// RUN: %boogie /monomorphize /useArrayTheory "%s" > "%t"
+// RUN: %boogie /lib /monomorphize /useArrayTheory "%s" > "%t"
```

Comment:
I think the Boogie test suite generally uses the `-` version of command-line options (instead of `/`).